jobs/wafs/stats/JEVS_WAFS_ATMOS_STATS

```diff
@@ -132,10 +145,16 @@ for resolution in $resolutions ; do
 	cat $file >> $finalfile
     done
     if [[ -s $finalfile ]] ; then
-	awk '!seen[$0]++' $finalfile > $COMOUTfinal/$finalfile
+	awk '!seen[$0]++' $finalfile > $STATSOUTfinal/$finalfile
     fi
 done
 
+if [ $SENDCOM = YES ] ; then
+    mv $STATSOUTfinal/* $COMOUTfinal/.
+    # COMOUTsmall
+    mv $STATSOUTsmall/* $COMOUTsmall/.
+fi
+
```

Comment:
I'm not entirely sure if this SENDCOM stuff can be in the J-job or needs to be in the exevs script.  @AliciaBentley-NOAA what do you think here?

Reply:
@YaliMao-NOAA this needs to be resolved now.  Code freeze is happening now, so this issue needs to be resolved.  Can you tend to it?